src/app/shared/components/cart-widget/cart-widget.component.scss

```diff
@@ -0,0 +1,19 @@
+.cart {
+    @apply p-2 border-gray-200 rounded-lg text-center cursor-pointer;
+
+    @apply md:border-2;
+    @apply hover:text-gray-800 hover:bg-gray-100 transition-colors;
+}
+
+.cart-image {
+    @apply w-8 h-8 inline-block;
+    @apply text-indigo-500;
+
+    @apply md:mb-1 md:mx-auto md:block;
+}
+
+.cart-contents {
+    @apply text-gray-900 inline;
+
+    @apply md:font-medium md:text-xl;
+}
```

Comment:
This is now identical to the wallet widget (along with most of the HTML). Should we abstract this down into shared widget component?

Reply:
that's a good idea. let's see if we figure out a solution